### DISCUSSION TITLE
香菜不是人吃的東西

### DISCUSSION THREAD

**Discussion by Superlan8** [1101-GS4719-C/Python-Community-GS4719-C#1184] [General]
今天晚餐吃酸辣粉忘了告訴店員不要加香菜，整個晚餐的過程非常痛苦，所以就想到來打一下這篇文分享一下討厭香菜的原因，也希望提醒我自己以後酸辣粉記得不要加香菜。

根據研究，在第11條染色體中有個用來感受氣味的基因OR6A2，這種基因對於醛類物質十分敏感，而香菜就蘊含大量的醛類物質，因此擁有這種基因的人就會特別討厭香菜。

此外，根據統計，討厭香菜的比例在世界各地也不盡相同，歐洲佔了17%，非洲占了14%，東亞佔了21%，但是在中東及東南亞這種在料理中常見香菜的國家，只有3%的人討厭他。由此可知，討厭香菜並不是只有靠遺傳物質決定的，文化及飲食習慣也影響很大。

最後，除了基因和文化外，討厭香菜還有一種原因，是因為香菜含有一種化學物質-戊二醛，這種物質被認為是一種臭味分子，對於某些人來說可能會讓人感到噁心或想吐。

**Comment by Lipunpun** [comment]
我還看過去酸辣粉店員放香菜，直接跟店員吵起來的，一直兇要店員換一碗（生氣傷身，大可不必），雖然店員問題也不小，但看他火氣那麼大，嚇到我只好多吃一點香菜。

**Comment by zilinng** [comment]
原來討厭香菜的原因有那麼多種！看來我也是擁有OR6A2基因的一員…前一陣子看到日本的必勝客推出香菜披薩,光用看的就全身起雞皮疙瘩….但喜歡吃香菜的朋友感覺都恨不得買機票去日本嘗試,只能說自己沒有口服,真的完完全全體會不到香菜到底好吃在哪了

**Comment by JohnnyXiao1024** [comment]
可惡，香菜看起來大逆風，身為沒有OR6A2基因的香菜愛好者，要來幫香菜說點話.....
香菜其實很讚啦，含有豐富的苦味素、硫化物等等，有助於降血脂，同時也照顧各位的心血管健康喔～

**Comment by CCello** [comment]
雖然香菜常常被大家討厭，可能是他的特殊氣味吧
但其實香菜含有豐富的營養物質，包括維生素C、維生素K和鈣，有助於保持身體健康和增強免疫力，並沒有大家想的這麼糟糕啦!

**Comment by ziihui** [comment]
香菜愛好者在這~~~><但我討厭芹菜!!!!
上網查資料後發現討厭芹菜跟香菜的理由一樣葉，也都是因為人體內都有一種叫OR6A2的基因~
順便補充冷知識
國際討厭香菜日是2/23
可惡怎麼可以沒有討厭芹菜日咧><

**Comment by pigeon9975** [comment]
除了香菜基因外，我查到還有一種叫TAS2R38的味覺基因也跟不敢吃菜有關耶！
人類有兩對TAS2R38基因，分別是AVI與PVI，其中，帶兩對PVI基因的人會對苦味極度敏感(約一般人的2.5倍)，可能連吃菜都會覺得苦之類的
好ㄟ終於找到合理挑食的理由了(誤

**Comment by WeiYen1103** [comment]
但我覺得香菜是一些食物中不可或缺的點綴!像是豬血糕，肉圓等等，有了香菜的加入才能讓整個食材的美味更加提升。

**Reply by Mitochondrionjessica** [reply]
同意! 還有花生冰淇淋春捲，香菜根本靈魂呀~

**Comment by eric202501** [comment]
對討厭食物的接受度是有可能隨著年齡成長改變，小時候真的無法入口，現在則是可以接受

**Reply by Mitochondrionjessica** [reply]
好特別~ 不知道是訓練出來的，還是長大味覺沒有小時候敏銳或是其他原因哈哈哈

**Comment by lyykey** [comment]
對於香菜我有相當的喜愛，獨特的氣味讓我覺得為食物增添幾分香氣。個人特愛豬血糕加上香菜，不知道為啥特別的香。
此外，其他香料、調味我也不會排斥，真不巧，剛好與版主意見不合了

**Comment by 20050201Shawn** [comment]
個人很喜愛香菜，喜愛到能生吃不配其他食物的程度。跟討厭香菜的朋友出去吃飯，幫他們吃香菜我也很開心。

**Comment by SusuTima** [comment]
個人覺得香菜加在蚵仔麵線裡就差不多了，其他東西加香菜感覺都怪怪的🤨。

**Comment by YANRONG0123** [comment]
我覺得香菜對我來說，沒有什麼問題，不會特別想吃，但也不會特別去挑掉，或是因此感到生氣，畢竟能夠讓味道改成另一個層次，也是個不錯的好方法，所以基本上除了辣椒以外，很多調味料我都盡量保持店家給的原樣。

**Comment by YifenLin** [comment]
我自己個人對香菜其實沒什麼意見，準確來說，我好像是一個不太挑食的人，很多大家排斥的食物我都可以接受，不過最近大家健康意識抬頭(絕對不是過年之後便太胖要減肥)，我如果去吃早餐店會跟阿姨說三明治不要加醬。

**Comment by 110101501** [comment]
我喜歡香菜，但香菜對我而言是一種佐料。凡是麵線羹、肉圓、花生捲冰淇淋等，配上香菜可是一大美味，讓原本食物的味道可以有更多不一樣的層次。
不過我不喜歡搞怪的吃法，像是配上披薩、或是刻意灌在腸衣裡變成香菜腸，就失去了香菜原為佐料的特性。

**Comment by LinChloe** [comment]
小時候完全沒想到會有人覺得調味用的菜(蔥薑蒜香菜)噁心。
直到上大學身邊愈來愈多人不吃這些，讓我有種我們以前不在同個世界嗎的感覺。
我是香菜加滿派的，但是必勝客的香菜pizza我真的點不下去。

**Comment by willychiuchiu** [comment]
香菜就真的很香，對我來說是不可或缺的佐料，如果像是酸辣粉、麵線這類食物中沒有加香菜，就像是食物沒有加鹽，雖然還是能吃，但就不會好吃！

**Comment by yxin214** [comment]
好有趣的文章！雖然看完文章後發現接受香菜與否還是和各國飲食文化有最大的關係，但我從來沒想過這居然和基因也有關係！看來以後同學不吃香菜時不能再說他挑食了哈哈哈。btw 肉圓+香菜、蚵仔麵線+香菜超好吃

**Comment by hbh10601** [comment]
香菜讚啦，米血糕、大腸麵線就是要加香菜才正統！

**Comment by buoioiry** [comment]
因為討厭香菜，又覺得香菜加在食物裡面的味道好重，如果有香菜加進去的料理都只聞得到他的味道就不太敢吃了

**Comment by Hahahahbla** [comment]
討厭香菜的原因包括遺傳、文化和化學物質影響。基因OR6A2使感受醛類物質敏感，而香菜含有大量醛類物質，導致特別討厭香菜的人。此外，世界各地文化和飲食習慣也影響人對香菜的態度。最後，戊二醛的臭味可能引起噁心感。我也討厭香菜: (

**Comment by wanka2030** [comment]
總覺得蔥 薑 蒜 九層塔 香菜 這些辛香料裡面

最多人無法接受的應該就是香菜了

但我本人最喜歡的就是香菜…

我跟我媽都是喜歡到可以直接單吃香菜加醬的那種XD

**Comment by jylin34** [comment]
我個人不排斥香菜，我很喜歡它的味道，加在各種小吃內都不錯！無論是鮮蝦餃子、炸春捲還是涼拌海鮮沙拉，只要加入適量的香菜，都能讓整道菜的風味更加豐富和令人回味。香菜所散發的清新香氣和微辣的風味，總是能夠在口中綻放出一種令人愉悅的感覺，讓我對它的喜愛更加濃厚。

**Comment by Eileen050824** [comment]
對我來說香菜是少量宜情，大量也是會有點噁心...我覺得香菜放在很多台灣小吃裡(大推花生捲冰淇淋！)，在味覺上有畫龍點睛的感覺，甚至它的存在很重要。但我也實在不能接受直接生吃一坨，或是商家推出的奇葩香菜食品(ex: 香菜棒棒糖、香菜豆花...)。

**Comment by yyujimmy** [comment]
文中提到的基因影響、文化差異和化學物質的影響，都是解釋討厭香菜現象的重要因素。這些因素之間的交互作用使得這個議題更加複雜和有趣。

我特別喜歡文章給出的統計數據，這些數據展示了討厭香菜的比例在不同地區的差異，這突顯了文化和飲食習慣對於飲食偏好的影響。

總的來說，這篇文章對於討厭香菜的原因進行了深入的分析，提供了豐富的信息和觀點，讓讀者更深入地了解這個問題。

**Comment by jim1743** [comment]
香菜對我來說就是佐料的一種，不會到特別討厭或特別喜歡，許多小吃加上香菜能夠讓食物的味道更上一層，但即使店家沒有加香菜，但對我來說也沒差，但我也很慶幸自己不是討厭香菜的人，否則就會一天到晚在意自己吃的食物是否加入了香菜。

**Comment by YOGA012** [comment]
我是一位香菜愛好者，但我也能理解有些人可能對香菜的味道不太喜歡，每個人的口味和喜好都不同，這是很正常的事情。對我來說，香菜的獨特香氣和味道是我喜歡的一部分，我覺得它可以為菜餚增添豐富的層次和風味。無論是在中餐還是西餐中，我都喜歡將香菜用作調味料，因為它能夠帶來獨特的口感和風味。所以，對於香菜愛好者來說，我們會繼續享受這種美味，並尊重每個人的口味偏好。

**Comment by cassy0201** [comment]
完全贊同哈哈哈，個人覺得香菜和芹菜味道一樣，都不喜歡一定要挑掉的那種，但身邊就有朋友是喜歡香菜討厭芹菜的，大家的口味真的都不一樣，接受度也不同

**Comment by 710388** [comment]
是嗎，我覺得還好本人不挑食，我覺得那些沒有很喜歡的東西還是要吃，反正都是養分，而且就算不喜歡也一下就感覺不到了

**Comment by NcuJerryKao** [comment]
根據研究，在第11條染色體中有個用來感受氣味的基因OR6A2，這種基因對於醛類物質十分敏感，而香菜就蘊含大量的醛類物質，因此擁有這種基因的人就會特別討厭香菜[[5](https://tw.news.yahoo.com/%E5%9C%8B%E9%9A%9B%E8%A8%8E%E5%8E%AD%E9%A6%99%E8%8F%9C%E6%97%A5-%E7%A0%94%E7%A9%B6%E6%8F%AD%E5%AF%86-%E5%96%9C%E6%AD%A1%E8%88%87%E5%90%A6%E7%94%B1%E5%9F%BA%E5%9B%A0%E6%B1%BA%E5%AE%9A-103240618.html)]。

此外，根據統計，討厭香菜的比例在世界各地也不盡相同，歐洲佔了17%，非洲佔了14%，東亞佔了21%，但是在中東及東南亞這種在料理中常見香菜的國家，只有3%的人討厭香菜[[2](https://www.womenshealthmag.com/tw/food-nutrition/diet/g43017903/cilantro-tastes-like-soap/)]。由此可知，討厭香菜並不是只有靠遺傳物質決定的，文化及飲食習慣也影響很大。

最後，除了基因和文化外，討厭香菜還有一種原因，是因為香菜含有一種化學物質-戊二醛，這種物質被認為是一種臭味分子，對於某些人來說可能會讓人感到噁心或想吐[[6](https://health.ettoday.net/news/1142883)]。

希望這些資訊能讓大家更了解討厭香菜的原因，也提醒自己以後記得提前告知店員不要加香菜。

**Comment by tshsu04224** [comment]
我也極度厭惡香菜🤮甚至點心麵有出香菜口味，真的太可怕了，我好像是不習慣各種偏提味用香料的味道，像是芹菜、八角之類的，都不太能接受，還有一個最討厭但不是香料的，就是三色豆，尤其是裡面的綠色豆豆，不管有多少，每次遇到一定先一顆顆挑乾淨才能下嚥

**Comment by YeeDaniel** [comment]
個人超愛香菜，我們家會煮蘿蔔排骨湯，原本已經很頂的湯頭，加入香菜，哇更上一層樓，真的超級棒！之前還有看到香菜披薩，很想試試看，可以考慮跟喜歡香菜的人一起share

**Comment by poiu6fi** [comment]
我自己對香菜沒什麼意見，加不加都沒關係，不過像是大腸麵線或是蘿蔔湯之類的東西，我還是覺得有加香菜味道才對
